acceptance/cluster/docker.go

```diff
@@ -97,19 +97,19 @@ func hasImage(l *LocalCluster, name, tag string) bool {
 	return false
 }
 
-func pullImage(l *LocalCluster, options types.ImagePullOptions) error {
+func pullImage(l *LocalCluster, ref string, options types.ImagePullOptions) error {
 	// Hack: on CircleCI, docker pulls the image on the first access from an
```

Comment:
i doubt it. this isn't a docker version upgrade, only a change in the client library.